CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# 0.6.2
```

Comment:
you can omit this changelog update. I realized that I usually end up writing these myself anyways, so I'll be removing that step from the PR checklist shortly...

Reply:
Reverted